lisa/sut_orchestrator/qemu/schema.py

```diff
@@ -15,10 +15,32 @@ class CloudInitSchema:
     extra_user_data: Optional[str] = None
 
 
+@dataclass_json()
+@dataclass
+class LibvirtHost:
+    address: Optional[str] = None
+    username: Optional[str] = None
+    private_key_file: Optional[str] = None
+
+    # The directory where lisa will store VM related files (such as disk images).
+    # This directory must already exist and the test user should have write permission
+    # to it.
+    lisa_working_dir: str = "/var/tmp"
```

Comment:
Is it possible to leave the default as the base VM image's directory for the local case?

Reply:
I guess it could be done if needed. Is it for backward compatibility?

Reply:
When running on a local dev box, it is nice to be able to easily see all the files created, particularly for cases where you keep the environment after the test completes/fails (e.g. while debugging).